docs/enhancements/on-prem-data-collection.md

```diff
@@ -0,0 +1,121 @@
+---
+title: on-prem-data-collection
+authors:
+  - "@CrystalChun"
+creation-date: 2022-09-30
+last-updated: 2022-09-30
+---
+
+# On-premises Data Collection
+
+## Summary
+
+To provide a better installation process and experience, the Assisted Installer ("AI") will collect data about installations when it is deployed on-premises as part of either [Red Hat Advanced Cluster Management for Kubernetes](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6) ("RHACM") or as a stand-alone Podman deployment. Data was already collected from clusters deployed on [Red Hat's Hybrid Cloud Console](https://console.redhat.com/openshift/assisted-installer/clusters) (“SaaS”). The collection of data from on-premises deployments will give the team additional insight into the product's usage, customers' pain points, and analytics for how to best improve the product.
+
+## Motivation
+
+Data helps us see how the product is being used, helps us to analyze trends, and to determine ways we can improve the product for our users. 
+
+### Goals
+
+1. Create a process to collect and send [event](https://github.com/openshift/assisted-service/tree/master/docs/dev/events.md) data from on-premises deployments of AI.
+2. Analyze and process on-premises data (by use of visualizations, dashboards, etc.) at Red Hat like SaaS data.
```

Comment:
all the data will be stored in the same ELK instance so we should be able to differentiate between SaaS and each on-prem environment

Reply:
Good point, added clarification! Does it sound ok?